openpype/settings/defaults/project_settings/global.json

```diff
@@ -591,6 +591,21 @@
                     "task_names": [],
                     "template_name": "simpleUnrealTextureHero"
                 }
+            ],
+            "transient_dir_profiles": [
+                {
+                    "hosts": [
+                        "nuke"
+                    ],
+                    "task_types": [],
+                    "task_names": [
+                        "transient_staging"
+                    ],
+                    "families": [],
+                    "subsets": [],
+                    "use_transient_dir": true,
+                    "transient_dir_persistent": true
+                }
             ]
```

Comment:
I don't think we'd need a default setting like this, right?

Reply:
Yeah, misfire.